enhancements/update/available-update-metadata.md

```diff
@@ -71,15 +74,16 @@ type Release struct {
   // +required
   Image string `json:"image"`
 
-  // URL describes the contents of this release and will include
-  // references to other resources as necessary.  Go here if you know
+  // url contains information about the release. Go here if you know
```

Comment:
Remove the `Go here if you know of no more-specific resource`.  It is not useful.

Reply:
> Remove the Go here if you know of no more-specific resource.

Dropped in d859e06 -> c49fab6 with notes in the `homepage` vs `url` section complaining about you forcing me to do this ;)

Reply:
And updated the Cincinnati-graph property definition to match the new first sentence with c49fab6 -> 12eff48.